### DISCUSSION TITLE
Changing working contracts over time

### DISCUSSION THREAD

**Discussion by nerdoc** [kimai/kimai#4901] [Questions & Answers]
I am evaluating Kimai for my company. How does it cope with changing working contracts?
E.g. when A team member changes he's working hours per week? Let's say 20 hours/week from Jan-Jun, (3 days) and 30 h/wk from Jul-Dec.(4 days)

You have to change the available holidays from 15 to 20 days/year (here in Austria), and calculate that appropriately per part times.
Does Kimai cope with that? I can see only "one" working contract / working hours page with no start/end date.

**Comment by kevinpapst** [comment]
You have to close (lockdown) previous months. 
That will write the data into extra storage.
Afterward you can change the expected (daily) hours, without the past being affected.
So you can change the contract with the start of every new month.

Holiday is not handled, you have to do that yourself by adding/removing amount via manual vacation bookings.
